spacy/tokens/doc.pyx

```diff
@@ -1361,11 +1361,9 @@ cdef class Doc:
     def to_dict(self, *, exclude=tuple()):
         """Export the document contents to a dictionary for serialization.
 
-        exclude (list): String names of serialization fields to exclude.
+        exclude (Iterable[str]): String names of serialization fields to exclude.
         RETURNS (bytes): A losslessly serialized copy of the `Doc`, including
             all annotations.
-
-        DOCS: https://spacy.io/api/doc#to_bytes
```

Comment:
This link was wrong, this method is currently not documented. Alternatively, we could go ahead and document it and fix this link. Not sure whether this needs to be in the public API though.

Reply:
I'm confused, this is part of the public API and is here: https://spacy.io/api/doc#to_bytes